pkg/retistruct/R/server.R

```diff
@@ -192,31 +256,34 @@ server <- function(input, output, session) {
   ## server, the actual click handling is in the plot click handler.
   observeEvent(input$add_tear, {
     set.state(state, 1)
-    set.status(output, "Click on the three points of the tear in any order. Double click to remove the latest point added and press cancel, to cancel.")
+    set.status(output, "Click on the three points of the tear in any order.
+                Double click to remove the latest point added and press cancel,
```

Comment:
Suggest replacing "added and press cancel, to cancel" with "Press cancel to abort adding the tear."
Also, I can crash the code by the following procedure:

- Open the Figure 1 demo
- Remove the left-hand tear
- Click on one point on the tear
- Double click on a point I've not added

It would be good to see if we could fix this behaviour.